cpp/src/DataStorm/Instance.cpp

```diff
@@ -1,31 +1,30 @@
 //
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
-#include <DataStorm/CallbackExecutor.h>
-#include <DataStorm/ConnectionManager.h>
-#include <DataStorm/Instance.h>
-#include <DataStorm/LookupI.h>
-#include <DataStorm/Node.h>
-#include <DataStorm/NodeI.h>
-#include <DataStorm/NodeSessionManager.h>
-#include <DataStorm/Timer.h>
-#include <DataStorm/TopicFactoryI.h>
-#include <DataStorm/TraceUtil.h>
-
-#include <IceUtil/UUID.h>
+
+#include "Instance.h"
+#include "CallbackExecutor.h"
+#include "ConnectionManager.h"
+#include "DataStorm/Node.h"
+#include "Ice/Timer.h"
+#include "Ice/UUID.h"
+#include "LookupI.h"
+#include "NodeI.h"
+#include "NodeSessionManager.h"
+#include "TopicFactoryI.h"
+#include "TraceUtil.h"
 
 using namespace std;
 using namespace DataStormI;
 
-Instance::Instance(const shared_ptr<Ice::Communicator>& communicator) : _communicator(communicator), _shutdown(false)
+Instance::Instance(const Ice::CommunicatorPtr& communicator) : _communicator(communicator), _shutdown(false)
 {
-    shared_ptr<Ice::Properties> properties = _communicator->getProperties();
+    Ice::PropertiesPtr properties = _communicator->getProperties();
 
     if (properties->getPropertyAsIntWithDefault("DataStorm.Node.Server.Enabled", 1) > 0)
     {
         properties->setProperty("DataStorm.Node.Adapters.Server.ThreadPool.SizeMax", "1");
```

Comment:
Not clear if we need to set Max dispatches instead of SizeMax with Ice 3.8?

Reply:
Why do we set SizeMax anyway? Are we using AMD or is everything sync?